CLIENT-SPECIFICATION.md

```diff
@@ -31,23 +31,23 @@ This section describes the standardised command-line interface (CLI) for clients
 
 ### Arguments
 
-A number of command-line options MUST be supported (unless otherwise specified) if a CLI is implemented:
+Several command-line options MUST be supported (unless otherwise specified) if a CLI is implemented:
 
 Option             | Required?   | Meaning
 -------------------|-------------|----------
 `-v`, `--version`  | Yes         | Shows the current version of the client, and the version of this specification that it implements.
 `-p`, `--platform` | Yes         | Specifies the platform to be used to perform the action (either listing or searching) as an argument. If this option is specified, the selected platform MUST be checked first instead of the current platform as described below.
-`-u`, `--update`   | Conditional | Updates the offline cache of pages. MUST be implemented if cache is supported.
+`-u`, `--update`   | Conditional | Updates the offline cache of pages. MUST be implemented if the cache is supported.
 `-l`, `--list`     | No          | Lists all the pages in the current platform to the standard output.
 `-L`, `--language` | No          | Specifies the preferred language for the page returned. Overrides other language detection mechanisms. See the [language section](#language) for more information.
 
-Clients MUST implement both the short and long version of an option.
+Clients MUST implement both the short and long versions of an option.
 
-Additional decoration MAY be printed if the standard output is a [TTY](http://www.linusakesson.net/programming/tty/index.php). If not, then the output MUST not contain any additional decorations. For example a page list MUST be formatted with 1 page name per line (to enable easy manipulation using standard CLI tools such as `grep` etc.).
+Additional decoration MAY be printed if the standard output is a [TTY](http://www.linusakesson.net/programming/tty/index.php). If not, then the output MUST not contain any additional decorations. For example, a page list MUST be formatted with a 1-page name per line (to enable easy manipulation using standard CLI tools such as `grep` etc.).
```

Comment:
I agree with the comma, but "1-page name" is incorrect, as it implies a name that occupies one page (whatever that would be 😅). What we want to say here is to have "one page-name" (abusing the hyphen to make my point across). Here's what I'd suggest:


```suggestion
Additional decoration MAY be printed if the standard output is a [TTY](http://www.linusakesson.net/programming/tty/index.php). If not, then the output MUST not contain any additional decorations. For example, a page list MUST be formatted with one page name per line (to enable easy manipulation using standard CLI tools such as `grep` etc.).
```